### ISSUE TITLE
[Feature Request]add parameter defualt_tracker for set_session

### ISSUE THREAD

**Issue by zero-custom** [trim21/transmission-rpc#240]
failed to set default trackers by set_session
return None and no errors was reported, but it just failed.

```
from transmission_rpc import Client;
print(Client().get_session().default_trackers);
Client().set_session(default_trackers=['http://any.other.tracker:6969/announce']);
print(Client().get_session().default_trackers)
```
output:
```
http://the.same.tracker:6969/announce
http://the.same.tracker:6969/announce
```
expect output:
```
http://the.same.tracker:6969/announce
http://any.other.tracker:6969/announce
```

**Comment by trim21** [comment]
good catch

**Comment by trim21** [comment]
will be release with 4.1.0

**Comment by zero-custom** [comment]
report: updated by pypi, and the latest version still not work.

![QQ截图20230227004805](https://user-images.githubusercontent.com/22925474/221424273-3c9e07e5-27d1-478f-b97b-64abc94ed785.png)

**Comment by trim21** [comment]
This doesn't look like a bug of this package, it looks like a bug of transmission daemon...

**Comment by trim21** [comment]
fixed in v4.1.1
